instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -304,31 +336,53 @@ def collect_request_attributes(scope):
         if isinstance(query_string, bytes):
             query_string = query_string.decode("utf8")
         http_url += "?" + urllib.parse.unquote(query_string)
+    result = {}
+
+    scheme = scope.get("scheme")
+    if scheme:
+        _set_http_scheme(result, scheme, sem_conv_opt_in_mode)
+    if server_host:
+        _set_http_host(result, server_host, sem_conv_opt_in_mode)
```

Comment:
Do we also have to set `_set_http_net_host(result, host, sem_conv_opt_in_mode)` here?

Reply:
Does not seem like `net.host.name` was supported previously in asgi instrumentation with old semconv. New semconv map it to `server.address` which we already populate.